core/templates/pages/splash-page/splash-page.component.rtl.css

```diff
@@ -0,0 +1,556 @@
+.splash-page .image-mobile {
+    display: none;
```

Comment:
Is it possible to set the RTL CSS generator so that these are indented by 2 not 4?

Also, can the RTL CSS generator put a comment at the top of RTL CSS files saying  "These files are autogenerated, please do not modify"? Otherwise, they'll get modified.

Reply:
Oh, the indentation was because the source file was indented wrongly when I created this file. I fixed the source indentation later but didn't regenerate this, will fix this. RTLCSS does not affect indentation, so whatever the format is present in source will be present in the output as well.
I'll check if such automatic commenting is possible.

Reply:
Done